Reports/report-docx-criticalAndHighIncidentsReport.json

```diff
@@ -0,0 +1,374 @@
+{
+  "id": "docxCriticalAndHighIncidentsReport",
+  "name": "Critical and High incidents",
+  "description": "All critical and high severity incidents that may need the analyst attention.",
```

Comment:
please add to all descriptions:
(DOC format).

Like with the CSV:
![image](https://cloud.githubusercontent.com/assets/5429033/25244378/24275dd0-2609-11e7-9550-c46e73b75d97.png)